src/main/java/sirius/kernel/xml/XMLCall.java

```diff
@@ -158,7 +158,7 @@ private InputStream getInputStream() throws IOException {
         return outcall.getResponse().body();
     }
 
-    private void logRequest(String response) throws IOException {
+    protected void logRequest(String response) throws IOException {
```

Comment:
shoudln't be needed....

Reply:
As we want to overwrite the `getInputStream` method, we must call logRequest there manually. We can create a new protected `getResponseBody()` method, in which we do the transformation in special cases (GEVIS), but I don't see a benefit in adding an additional method that returns yet another InputStream in comparison with the approach in https://github.com/scireum/sirius-kernel/pull/446/files . We would also lose the benefit of logging the raw responses as we would need to transform them before logging.